pkg/v1/tkg/web/src/app/views/landing/aws-wizard/node-setting-step/node-setting-step.component.ts

```diff
@@ -564,6 +565,12 @@ export class NodeSettingStepComponent extends StepFormDirective implements OnIni
         }
     }
 
+    // Because we use clusterTypeDescriptor in our description, we need to trigger a step description change whenever
```

Comment:
This all works really well. Maybe a separate PR but it would be very helpful I think if we enhance some of the docs for our methods to include some kind of breadcrumb trail. For example, something as simple as "See StepFormDirective.setClusterTypeDescriptor for usage." That might not be a great example but it would help us immediately go to StepFormDirective setClusterTypeDescriptor method to see where 'onChangeClusterTypeDescriptor' is being called, and then it's easy to work back to see that this is triggered by a branding change event. Maybe this type of exercise would be good to have Raymond and myself take on to get us more familiar with the refactored code. Curious to hear your thoughts on this.